### PR TITLE
NOISSUE - Remove redundant error check

### PR DESCRIPTION
**Pull request by dborovcanin**
Signed-off-by: Dušan Borovčanin <dusan.borovcanin@mainflux.com>

### CODE REVIEW

File: writers/influxdb/influxdb.go [hunk 1]
```diff
@@ -41,10 +41,7 @@ func (repo *influxRepo) Save(msg mainflux.Message) error {
 	}
 
 	bp.AddPoint(pt)
-	if err := repo.client.Write(bp); err != nil {
-		return err
```
Comment (anovakovic01) [inline writers/influxdb/influxdb.go:45]: Maybe add public error?

**Review by anovakovic01 — DISMISSED** [review]

**Review by drasko — COMMENTED** [review]
Looks OK on first look, I think we can accept

File: writers/influxdb/influxdb.go (hunk 1 above)
Comment (dborovcanin) [reply writers/influxdb/influxdb.go:45]: We could. @mijicd Suggestions? Currently, I see no benefits of doing that, but I don't mind exporting an error, as well.

Comment (mijicd) [reply writers/influxdb/influxdb.go:45]: There's no point of adding that.

**Review by mijicd — APPROVED** [review]

**Pull request merged**